docs/content/dagster-plus/account/managing-users/managing-user-roles-permissions.mdx

```diff
@@ -14,32 +14,56 @@ In this guide, we'll cover how RBAC works in Dagster+, how to assign roles to us
 
 ## Dagster+ user roles
 
-Dagster+ uses a hierarchical model for RBAC, meaning that the most permissive roles include permissions from the roles beneath them. The following user roles are currently supported, in order from the **most** permissive to the **least** permissive:
+Dagster+ employs a flexible approach to user roles and permissions. This system is built on two fundamental concepts:
 
-- Organization Admin
-- Admin
-- Editor
-- Launcher (Pro plans only)
-- Viewer
-
-For example, the **Admin** user role includes permissions specific to this role and all permissions in the **Editor**, **Launcher**, and **Viewer** user roles. Refer to the [User permissions reference](#user-permissions-reference) for the full list of user permissions in Dagster+.
+1. **Permission Scope** - Permissions are context-specific, falling into two main categories:
+   - Organization-wide settings (for example, "Create Teams")
+   - Deployment-specific actions (for example, "Launch and Cancel Backfills")
+2. **Role Types** - Dagster+ supports two types of roles:
+   - **Default Roles:** Hierarchical roles, based on sensible defaults that come out of the box with Dagster.
+   - **Custom Roles:** Allows you to define tailored roles with specific sets of permissions
 
+This dual approach provides both the simplicity of predefined roles and the flexibility to create custom roles that precisely match your organization's needs.
 ### User role enforcement
 
-All user roles are enforced both in Dagster+ and the [GraphQL API](/concepts/webserver/graphql).
+All roles are enforced both in Dagster+ and the [GraphQL API](/concepts/webserver/graphql).
 
 ### Teams
 
 Dagster+ Pro users can create teams of users and assign default permission sets. Refer to the [Managing teams in Dagster+](/dagster-plus/account/managing-users/managing-teams) guide for more info.
 
+--- 
+## Creating and editing custom roles
```

Comment:
I would make "Creating" and "Editing" two separate sections and go ahead and repeat the first two steps in each, to make each task a bit easier to parse.